### ISSUE TITLE
Mail Notifications URL error

### ISSUE THREAD

**Issue by eddiepiazza** [humhub/humhub#2562]
Hi, I am getting error by accessing to the url of notifications sent by mail.
![image](https://cloud.githubusercontent.com/assets/8577866/25956192/f83cf2a6-3662-11e7-98ae-8e744d3353ae.png)

![image](https://cloud.githubusercontent.com/assets/8577866/25956215/0857caee-3663-11e7-8043-2719cadbedb0.png)

The URL seems correct: `https://www.gropedia.com/notification/entry?id=616` but seems the database is not recognising the entry.

Any solution or diagnosis? 

Than you very much.

**Comment by buddh4** [comment]
Are you sure you are not logged in with another account? You have to be logged in with the receiver account.

**Comment by eddiepiazza** [comment]
Yes I was logged, I made some tests and I discover that the problem is related with cache (maybe). Because I updated the page (CTRL+F5) and the posts were deleted. So it´s normal that was impossible to request the notification but here is more a problem of sync.

My recommendation is to include a message "Post deleted from the user" instead of delete the entry ID from the database and Show the actual notification.
